GitCommands/UserRepositoryHistory/RecentRepoInfo.cs

```diff
@@ -44,6 +48,30 @@ public RecentRepoInfo(Repository repo, bool mostRecent)
         public override string ToString() => Repo.ToString();
     }
 
+    public enum RepoType
+    {
+        Common,
+        Current,
+        Deleted,
+
+        Submodule,
+        SubmoduleRevisionUpDirty,
+        SubmoduleRevisionUp,
+        SubmoduleRevisionDownDirty,
+        SubmoduleRevisionDown,
+        SubmoduleRevisionSemiUpDirty,
+        SubmoduleRevisionSemiUp,
+        SubmoduleRevisionSemiDownDirty,
+        SubmoduleRevisionSemiDown,
+        SubmoduleDirty,
+        FileStatusModified,
+
+        Superproject,
```

Comment:
Topmodule is used elsewhere for the top project/module and mostly what the Git documentation uses (unclear)
Superproject is just next level

Reply:
It's just the next level in the current implementation:

![image](https://user-images.githubusercontent.com/1150330/103041150-9d2f7780-4586-11eb-881e-873045e44379.png)

It's calculated relative to the currently opened project. Is it incorrect?

Reply:
If it is only the next upper level, then superproject is correct name
Still not sure what icon I want...